src/Homework1/program10.py

```diff
@@ -0,0 +1,21 @@
+# 10 lines: Time, conditionals, from..import, for..else
+
+
+from time import localtime
```

Comment:
here you have 1 line between imports and code

Reply:
Left 1 line beetween import and code in all files